examples/reference/ambientLight/ambientLight1/ambientLight1.rpde

```diff
@@ -0,0 +1,18 @@
+P3D <- "processing.opengl.PGraphics3D"
+PI <- pi
```

Comment:
I think it is not what we need. In GSoC we will support the variables so it is only temporary. Both pi and 3.1415 are fine, IMO.